spec/controllers/exercises_controller_spec.rb

```diff
@@ -288,95 +288,6 @@
     end
   end
 
-  describe 'POST #submit' do
```

Comment:
Rather than removing this test, we could consider transforming it for our new `SubmissionsController#finalize` method.

Reply:
Reused in `score_spec.rb`